pamelalozano/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,31 @@
+
+package com.google.sps.servlets;
+
+
+import java.util.*;  
+import com.google.gson.Gson;
+import java.io.IOException;
+
+public class Comment {
+  
+  private String subject;
+  private String msg;
+
+  public Comment(String s, String m) {
```

Comment:
keeping these as full words (subject, message) improves readability of the code

Reply:
Okay done!